libraries/SPIFFS/examples/SPIFFS_Test/SPIFFS_Test.ino

```diff
@@ -1,6 +1,11 @@
 #include "FS.h"
 #include "SPIFFS.h"
 
+// Format SPIFFS the first time you run a test
+// or else use the SPIFFS plugin to create a partition
+// https://github.com/me-no-dev/arduino-esp32fs-plugin
+//#define FORMAT_SPIFFS true
```

Comment:
uncomment this define :) tests are failing. Maybe rename to `FORMAT_SPIFFS_IF_FAILED` to be more clear when it will be formatted.